sales.html

```diff
@@ -5,20 +5,21 @@
     <title>All Stores Sales Data</title>
   </head>
   <body>
-    <h1>Pike</h1>
-    <ul id="pike">
-    </ul>
+    <h1>All Store Data</h1>
+    <h2>Pike</h2 class="heading">
```

Comment:
This h2 got put in the wrong spot, but this is a super clean way to be building things!!